build.gradle

```diff
@@ -106,6 +106,10 @@ subprojects {
         apply plugin: 'com.android.library'
     }
     apply plugin: 'kotlin-android'
+    if (name != 'analytics' && name != 'image') {
+        // Most modules make use of the synthetic accessors; ':db' uses the experimental 'Parcelize' annotation, so it does some configuration beyond that.
+        apply plugin: 'kotlin-android-extensions'
```

Comment:
i know these two don't need it but is there harm in applying them? looking at this from a simplification stand point

Reply:
Slightly, yes. Since I'm
1. trying to avoid changing functionality here
2. still not confident I know precisely what the plugin does

I scheduled that for a subsequent PR. For a point of reference, the first time you apply the kapt plugin to a module you add at least .5 seconds to the configuration time.